components/form/FCreditCardInput.vue

```diff
@@ -71,8 +71,8 @@ import FCreditCardIcon from '@/components/FCreditCardIcon.vue';
 import { computed, ref, watch } from 'vue';
 import { useFieldWithValidation } from '@/composables/useFieldWithValidation';
 import { useInputEventBindings } from '@/composables/useInputEventBindings';
-import getCardType from 'credit-card-type';
-import { CreditCardTypeCardBrandId } from 'credit-card-type/dist/types';
+import type { CreditCardInformation } from '@/modules/credit-card-types';
+import { getCardInformation } from '@/modules/credit-card-types';
```

Comment:
1) The module file could itself be called `'@/modules/credit-cards'`

2) `Info` is a completely valid word in English, and as much shorter, maybe better to use
```suggestion
import type { CreditCardInfo } from '@/modules/credit-card-types';
import { getCardInfo } from '@/modules/credit-card-types';
```